framework/src/multiapps/TransientMultiApp.C

```diff
@@ -260,7 +254,10 @@ TransientMultiApp::solveStep(Real dt, Real target_time, bool auto_advance)
           _failures++;
 
           if (_failures > _max_failures)
-            mooseError("While sub_cycling "<<_name<<_first_local_app+i<<" REALLY failed!"<<std::endl);
+          {
+            mooseWarning("While sub_cycling "<<_name<<_first_local_app+i<<" REALLY failed!"<<std::endl);
+            return false;
```

Comment:
I have no problem fixing this... but if we're going to make it a "thing" then we should test for it...